core/test/Test/Pos/Core/Gen.hs

```diff
@@ -425,7 +431,7 @@ genChainDifficulty = ChainDifficulty <$> genBlockCount
 
 genCoeff :: Gen Coeff
 genCoeff = do
-    integer <- Gen.integral (Range.constant 0 10)
+    integer <- Gen.integral (Range.constant 1 10)
```

Comment:
Since this was modified it came to my attention - is there a reason we use such a small range?

I think that `Data.Fixed` supports a big range - could you check up on this?

Reply:
Yep we should have been using a range from 0 to 1000000000000 because of  `Nano` like we discussed.  https://github.com/input-output-hk/cardano-sl/blob/0cad2c33c7d2973e972fbe295c1a87f2b831a70a/core/src/Pos/Core/Common/Coeff.hs#L14